src/ctapipe/tools/calculate_pixel_stats.py

```diff
@@ -121,6 +121,15 @@ def start(self):
                 instrument=False,
                 pointing=False,
             )[tel_id]
+            # Check if the chunk size does not exceed the table length of the input data
+            if self.stats_calculator.stats_aggregators[
+                self.stats_calculator.stats_aggregator_type.tel[tel_id]
+            ].chunk_size > len(dl1_table):
+                raise ToolConfigurationError(
+                    f"Change --StatisticsAggregator.chunk_size to decrease the chunk size "
+                    f"of the aggregation to at least '{len(dl1_table)}' (table length of the "
```

Comment:
Despite the leading 'decrease' -  'at least' could be a bit ambiguous here. Maybe ...' to a maximum of' .. would be more clear.